app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import time
+from datetime import datetime
+
+
+def create_file_every_second():
+    while True:
+        file_name = f"app-{datetime.now().hour}_{datetime.now().minute}_" \
+                    f"{datetime.now().second}.log"
+        file_content = datetime.now()
```

Comment:
If you want to name this variable as `file_content` it's better to apply str function to it in this line